web/beacon-app/src/features/projects/components/EditProject/EditProjectModal.tsx

```diff
@@ -44,7 +44,6 @@ function RenameProjectModal({ open, handleModalClose, project }: ChangeRoleModal
     <Modal
       open={open}
       title="Edit Project"
-      containerClassName="overflow-scroll  max-w-[80vw] lg:max-w-[50vw] no-scrollbar"
```

Comment:
This modal appears to look a little more narrow compared to how it currently looks. 

![Screenshot 2023-07-26 at 12 30 42 PM (2)](https://github.com/rotationalio/ensign/assets/94616884/fbdb15c5-4964-477a-a126-9afccbaf00af)